### PR TITLE
Added derrivative of sigmoid

### LINKED ISSUE #60: Convert Activations from simple functions to classes and add gradient method.
Currently, all the activations in activations.py are simple functions. 

However, for future implementation of Neural Networks, we will also need derivatives methods of each of these
functions. You can have a look into loss_func.py for reference. There, each class represents a loss function and has both loss
and derivative methods. You have to implement something similar for activations.

### PR DESCRIPTION
**Pull request by Abjcodes**
#### References to other Issues or PRs or Relevant literature

Fixes #60 

#### Brief description of what is fixed or changed
Created Sigmoid class with activation and derivative methods. The derivative method returns the derivative of sigmoid of the vector X.

#### Other comments

### CODE REVIEW

**Review by rohansingh9001 — APPROVED** [review]
LGTM and Merged
